backend/src/Service/Order/OrderService.php

```diff
@@ -111,13 +111,24 @@ public function getSpecificOrderForStore(int $id): ?OrdersResponse
         $order = $this->orderManager->getSpecificOrderForStore($id);
         if($order) {
             
+            $order['attention'] = null;
```

Comment:
I think this line isn't necessary as long as you have already defined the attention var in the response class that it could be null.

Reply:
It does not return within the response so it must be defined or it will disappear if it is empty